samcli/local/docker/container.py

```diff
@@ -469,6 +469,7 @@ def _write_container_output(
 
     @staticmethod
     def _handle_data_writing(output_stream: Union[StreamWriter, io.BytesIO, io.TextIOWrapper], output_data: bytes):
+        output_data = output_data.decode("utf-8").replace("\r", os.linesep).encode("utf-8")
```

Comment:
Might add a comment here which points to this PR for details. Also, did we confirm this behavior on Windows?

Reply:
Yes, I validated on Windows. I can add a comment.